bot/exts/valentines/valentine_zodiac.py

```diff
@@ -13,29 +15,115 @@
 LETTER_EMOJI = ':love_letter:'
 HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
 
+ZODIAC_SIGNS = ["Aries", "Taurus", "Gemini", "Cancer", "Leo", "Virgo", "Libra",
+                "Scorpio", "Sagittarius", "Capricorn", "Aquarius", "Pisces"]
+
+MONTH_NAME = {"January": 1, "Jan": 1, "Feburary": 2, "Feb": 2, "March": 3, "Mar": 3,
+              "April": 4, "Apr": 4, "May": 5, "June": 6, "Jun": 6, "July": 7, "Jul": 7,
+              "August": 8, "Aug": 8, "September": 9, "Sept": 9, "October": 10, "Oct": 10,
+              "November": 11, "Nove": 11, "December": 12, "December": 12
+              }
+
 
 class ValentineZodiac(commands.Cog):
     """A Cog that returns a counter compatible zodiac sign to the given user's zodiac sign."""
 
     def __init__(self, bot: commands.Bot):
         self.bot = bot
-        self.zodiacs = self.load_json()
+        self.zodiacs, self.zodiac_fact = self.load_comp_json()
 
     @staticmethod
-    def load_json() -> dict:
+    def load_comp_json() -> dict:
         """Load zodiac compatibility from static JSON resource."""
+        p1 = Path("bot/resources/valentines/zodiac_explanation.json")
         p = Path("bot/resources/valentines/zodiac_compatibility.json")
```

Comment:
Why not name these `explanation_file` and `compatibility_file` respectively? `p1` and `p` seems too abstract and inconsistent.
```suggestion
        explanation_file = Path("bot/resources/valentines/zodiac_explanation.json")
        compatibility_file = Path("bot/resources/valentines/zodiac_compatibility.json")
```